README.md

```diff
@@ -33,7 +33,7 @@ Check it out these [cool demos](http://react-responsive-carousel.js.org/storyboo
 Customize it yourself <https://codepen.io/leandrowd/pen/xLvOPd>
 
 ### Installing as a package
-`npm install react-responsive-carousel --save`
+`$ npm install react-responsive-carousel --save`
```

Comment:
Hi @M-ZubairAhmed, thanks for your contribution.

I don't agree with the usage of `$` before shell commands.  It makes it harder to copy and paste commands before running on the terminal.

Reply:
oh ok, what do you want me to do with this PR?

Reply:
You can decline it. Feel free to submit other changes if you want :)